src/main/java/app/controllers/OrderController.java

```diff
@@ -44,6 +49,9 @@ private static void loadCupcakeParts(Context ctx, ConnectionPool connectionPool)
     }
 
     public static void addToCart(Context ctx, ConnectionPool connectionPool) throws DatabaseException {
+
+        ctx.sessionAttribute("currentUser");
```

Comment:
Current user is acquired but never used. It can be removed if it is not being used